backend/src/routes/api/accelerator-profiles/acceleratorProfilesUtils.ts

```diff
@@ -10,13 +10,15 @@ export const postAcceleratorProfile = async (
 ): Promise<{ success: boolean; error: string }> => {
   const { customObjectsApi } = fastify.kube;
   const { namespace } = fastify.kube;
-  const body = request.body as AcceleratorProfileKind['spec'];
+  const body = request.body as { name?: string } & AcceleratorProfileKind['spec'];
+  const { name } = body;
+  delete body.name;
```

Comment:
Nit
```suggestion
  const requestBody = request.body as { name?: string } & AcceleratorProfileKind['spec'];
  const { name, ...body } = requestBody;
```
There really isn't a need to delete

Reply:
Leaving it would create/modify a `name` field on the `spec`

Reply:
`...body` is the remaining properties of the request body -- which leaves you with a body object the same as you had before you added a new `name` property, right? I didn't test this, but I compared it against the code you originally had before your change... I can actually test this...

Reply:
Looks fine to me... 
<img width="1175" alt="image" src="https://github.com/user-attachments/assets/acdc353f-e637-407b-9931-763ac94659e5">

Did you run into an issue?

Reply:
Nope, works fine. My bad. Updated.